README.md

```diff
@@ -14,6 +9,32 @@ Count and sum the scores of a bowling game for one player. For this challenge, y
 
 A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
 
+USER STORIES
```

Comment:
Fantastic breakdown of the user stories! I got really stuck when i tried to break it down.